includes/admin.form.inc

```diff
@@ -351,22 +354,1231 @@ function islandora_scholar_admin_pdf_form($form, &$form_state) {
+    '#states' => array(
+      'visible' => array(
+        ':input[name="islandora_scholar_xpaths_enabled"]' => array('checked' => TRUE),
+      ),
+    ),
+    '#ajax' => array(
+      'callback' => 'islandora_scholar_xpaths_citeproc_callback',
+      'wrapper' => 'islandora_scholar_xpaths_citeproc_title',
+    ),
+   '#field_suffix' => (!empty($form_state['values']['islandora_scholar_xpaths_demo']) ? _mods_id($form_state['values']['islandora_scholar_xpaths_local_container-author'], check_plain($form_state['values']['islandora_scholar_xpaths_demo'])) : t("Modify to refresh value (PID for demo object is required).")),
+  ));
+  $form['islandora_scholar_xpaths_citeproc'] += _islandora_scholar_add_variable('islandora_scholar_xpaths_local_collection-editor', '//mods:mods[1]/mods:relatedItem[@type="series"]/mods:name', array(
+    '#type' => 'textfield',
+    '#title' => t('Collection Editor'),
+    '#description' => t('Default Value: //mods:mods[1]/mods:relatedItem[@type="series"]/mods:name <br/>XPath to use for the collection-editor.'),
+    '#states' => array(
+      'visible' => array(
+        ':input[name="islandora_scholar_xpaths_enabled"]' => array('checked' => TRUE),
+      ),
+    ),
+    '#ajax' => array(
+      'callback' => 'islandora_scholar_xpaths_citeproc_callback',
+      'wrapper' => 'islandora_scholar_xpaths_citeproc_title',
+    ),
+   '#field_suffix' => (!empty($form_state['values']['islandora_scholar_xpaths_demo']) ? _mods_id($form_state['values']['islandora_scholar_xpaths_local_collection-editor'], check_plain($form_state['values']['islandora_scholar_xpaths_demo'])) : t("Modify to refresh value (PID for demo object is required).")),
+  ));
+  $form['islandora_scholar_xpaths_citeproc'] += _islandora_scholar_add_variable('islandora_scholar_xpaths_role_term', 'mods:role/mods:roleTerm', array(
+    '#type' => 'textfield',
+    '#title' => t('Role Term'),
+    '#description' => t('Default Value: mods:role/mods:roleTerm <br/>XPath to use for the role_term.'),
+    '#states' => array(
+      'visible' => array(
+        ':input[name="islandora_scholar_xpaths_enabled"]' => array('checked' => TRUE),
+      ),
+    ),
+    '#ajax' => array(
+      'callback' => 'islandora_scholar_xpaths_citeproc_callback',
+      'wrapper' => 'islandora_scholar_xpaths_citeproc_title',
+    ),
+   '#field_suffix' => (!empty($form_state['values']['islandora_scholar_xpaths_demo']) ? _mods_id($form_state['values']['islandora_scholar_xpaths_role_term'], check_plain($form_state['values']['islandora_scholar_xpaths_demo'])) : t("Modify to refresh value (PID for demo object is required).")),
+  ));
+  drupal_add_js("(function($) {
+    Drupal.behaviors.islandora_scholar = {
+      attach: function (context, settings) {
+      $('body').on('load', function(){
+        console.log('xxx');
+      });
+      }
+    };
+  })(jQuery);", 'inline');
+  module_load_include('inc
```

Comment:
Ideally, `$build_string` would also be populated using `t()`, as in
```php
        $build_string = t('For pid "@pid": <span style="color:blue">@mods_value_string</span>', array(
          '@pid' => $pid,
          '@mods_value_string' => $mods_value_string,
        ));
```
Similary for [L1567](https://github.com/Islandora/islandora_scholar/pull/299/files#diff-0cdd19abe14e51fe9e7f061ec72c86d3R1567), [L1570](https://github.com/Islandora/islandora_scholar/pull/299/files#diff-0cdd19abe14e51fe9e7f061ec72c86d3R1570) and [L1573](https://github.com/Islandora/islandora_scholar/pull/299/files#diff-0cdd19abe14e51fe9e7f061ec72c86d3R1573) above.